index.html

```diff
@@ -0,0 +1,349 @@
+              <li>
+              <a href="#" class="social-network">
+                <i class="fa fa-linkedin"></i>
+              </a>
+              </li>
+              <li>
+              <a href="#" class="social-network">
+                <i class="fa fa-medium"></i>
+              </a>
+              </li>
+              <li>
+              <a href="#" class="social-network">
+                <i class="fa fa-github"></i>
+              </a>
+              </li>
+              <li>
+              <a href="#" class="social-network">
+                              <i class="fa fa-angellist"></i>
+              </a>
+              </li>
+              </ul>                
+          </div>
+          <div class="action">
+            <button type="button" class="btn about-me-btn">Get my resume </button>
+          </div>
+        </div>
+        <ul class="skill-list">
+            <!----<div class="frame-12"> --->
+            <li class="frame-7-languages">
+              <a href="#">Languages</a>
+              <i class='fa fa-chevron-down'></i>
+
+               <!---<ul class="language-list"> ---> 
+              <li>
+                <ul class="language-list">               
+                  <li class="frame-9-10-11">                           
+                      <img src="./images/Ellipse-1.png" alt="Ellipse 1(Javascript)">
+                      <a href="#">Javascript</a>                
+                  </li>   
+                  <li class="frame-9-10-11">                    
+                      <img src="./images/ellipse-2.png" alt="Ellipse 1(HTML)">
+                      <a href="#">HTML</a>                             
+                  </li>                         
+                  <li class="frame-9-10-11">               
+                      <img src="./images/ellipse-3.png" alt="Ellipse 3(CSS)">
+                      <a href="#">CSS</a>                    
+                  </li>
+                </ul>
+              </li>
+            
+            <li class="skill-and-framework">
+              <a href="#">Frameworks</a>
+              <i class='fa fa-chevron-right'></i>
+            </li>
+            <li class="skill-and-framework">
+              <a href="#">Skills</a>
+              <i class='fa fa-chevron-right'></i> 
+            </li>
+        </ul>
+      </div>           
+
+    </section>
+    <!---------End of about myself section-------->
+    <!---------End of mobile-finish-version----->
+
+
+    <!--------Form container mobile version------>
+    <footer>
+      <div class="form-container">
+        <img src="/images/contact-form-shapes-mobile.png" alt="Contact Form Shapes" class="contact-form-shapes">
+        <span class="contact-me-title">Contact me</span>
+        <p class="contact-me-text">
+          If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
```

Comment:
- [ ]  Kindly add the appropriate margins/padding to the contact section text contents to match the Figma design template chosen. Please check the Figma design carefully  😄 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/187533825-44aadf01-bce6-456e-8f06-bf97520f5e49.png))  |![image](https://user-images.githubusercontent.com/63473584/187534100-2bb41d47-9f65-4455-b11f-3bc7e6dc423f.png)
  |

Reply:
Hi Mr.
I used 103px padding-top and it is the same as that of Figma. I can add of course.